app/forms/teacher_interface/further_information_request_item_text_form.rb

```diff
@@ -6,7 +6,10 @@ class FurtherInformationRequestItemTextForm < BaseForm
     attribute :response, :string
 
     validates :further_information_request_item, presence: true
-    validates :response, presence: true
+    validates :response,
+              presence: {
+                message: "Please respond to the assessor's request",
+              }
```

Comment:
We should use the `I18n` locals instead.

Take a look at `teacher_interface.en.yml` file for other examples.

Reply:
You shouldn't have to make any changes here.